README.md

```diff
@@ -1,40 +1,33 @@
 
-- **Title:** Deep Learning Model Extension
-- **Identifier:** <https://schemas.stacspec.org/v1.0.0-beta.3/extensions/dl-model/json-schema/schema.json>
-- **Field Name Prefix:** dlm
+- **Title:** Model Extension
+- **Identifier:** [https://schemas.stacspec.org/v1.0.0-beta.3/extensions/ml-model/json-schema/schema.json](https://schemas.stacspec.org/v1.0.0-beta.3/extensions/ml-model/json-schema/schema.json)
+- **Field Name Prefix:** mlm
 - **Scope:** Item, Collection
-- **Extension [Maturity Classification][stac-ext-maturity]:** Proposal
-- **Owner**:
-  [@sfoucher](https://github.com/sfoucher)
-  [@fmigneault](https://github.com/fmigneault)
-  [@ymoisan](https://github.com/ymoisan)
-
-[stac-ext-maturity]: https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity
-
-This document explains the Template Extension to the [SpatioTemporal Asset Catalog][stac-spec] (STAC) specification.
-This document explains the fields of the STAC Deep Learning Model (dlm) Extension to a STAC Item. 
-The main objective is to be able to build model collections that can be searched 
-and that contain enough information to be able to deploy an inference service.
-When Deep Learning models are trained using satellite imagery, it is important
-to track essential information if you want to make them searchable and reusable:
-1. Input data origin and specifications
-2. Model base transforms
-3. Model output and its semantic interpretation
-4. Runtime environment to be able to run the model
+- **Extension Maturity Classification:** Proposal
+- **Owner:**
+  - [@sfoucher](https://github.com/sfoucher)
+  - [@fmigneault](https://github.com/fmigneault)
+  - [@ymoisan](https://github.com/ymoisan)
+  - [@rbavery](https://github.com/rbavery)
+
+This document explains the Template Extension to the [SpatioTemporal Asset Catalog (STAC)](https://github.com/radiantearth/stac-spec) specification. This document explains the fields of the STAC Model Extension to a STAC Item. The main objective of the extension is two-fold: 1) to enable building model collections that can be searched alongside associated STAC datasets and 2) to record all necessary parameters, artifact locations, and high level processing steps to deploy an inference service. Specifically, this extension records the following info to make ML models searchable and reusable:
+1. Sensor band specifications
+2. The two fundamental transforms on model inputs: rescale and normalization
+3. Model output shape, data type, and its semantic interpretation
+4. An optional, flexible description of the runtime environment to be able to run the model
 5. Scientific references
 
-[stac-spec]: https://github.com/radiantearth/stac-spec
+Note: The spec is biased towards supervised ML models the produce classifications. However, fields that relate to supervised ML are optional and users can use the fields they need for different tasks.
```

Comment:
much agreed here. I wonder how much we should focus on describing outputs in this extension. My feeling is that this could get very tricky very quickly with all the different kinds of models out there. But I think for the most popular tasks like regression, object detection, semantic segmentation, and instance segmentation, we should provide standards.

Reply:
I agree. Especially for EO related tasks interacting with some form of image bands stack, we can assume a few common output structures to simplify the definition. I think it would be sufficient to have the dimensions of the output tensor(s), their dtype and their names for identification purpose.